lib/gollum/app.rb

```diff
@@ -110,14 +111,17 @@ class App < Sinatra::Base
     end
 
     namespace '/gollum' do
+      
+      get '/assets/mathjax/*' do
+        env['PATH_INFO'].sub!("/#{Precious::Assets::ASSET_URL}/mathjax", '')
```

Comment:
Will `sub!` always succeed, even if mathjax is not enabled?

Reply:
Yes, it's only replacing out '/gollum/assets/mathjax' from the PATH_INFO. Come to think of it, I might just put in the string `'/gollum/assets/mathjax'` instead of `"/#{Precious::Assets::ASSET_URL}/mathjax"`?

Reply:
Sure, makes sense.